ansible/deploy-openstack-config.yml

```diff
@@ -155,16 +155,16 @@
         dest: '{{ src_directory }}/{{ kayobe_config_name }}/etc/kayobe/hooks/{{ item.dest }}'
         state: link
       loop:
-        - { src: growroot.yml, dest: overcloud-host-configure/pre.d/5-growroot.yml }
-        - { src: fix-hostname.yml, dest: overcloud-host-configure/pre.d/10-fix-hostname.yml }
-        - { src: fix-networking.yml, dest: overcloud-host-configure/pre.d/15-fix-networking.yml }
-        - { src: configure-vxlan.yml, dest: overcloud-host-configure/pre.d/20-configure-vxlan.yml }
-        - { src: growroot.yml, dest: seed-host-configure/pre.d/5-growroot.yml }
-        - { src: fix-networking.yml, dest: seed-host-configure/pre.d/15-fix-networking.yml }
-        - { src: configure-vxlan.yml, dest: seed-host-configure/pre.d/20-configure-vxlan.yml }
-        - { src: growroot.yml, dest: infra-vm-host-configure/pre.d/5-growroot.yml }
-        - { src: fix-networking.yml, dest: infra-vm-host-configure/pre.d/15-fix-networking.yml }
-        - { src: configure-vxlan.yml, dest: infra-vm-host-configure/pre.d/20-configure-vxlan.yml }
+        - { src: tools/growroot.yml, dest: overcloud-host-configure/pre.d/5-growroot.yml }
+        - { src: fixes/fix-hostname.yml, dest: overcloud-host-configure/pre.d/10-fix-hostname.yml }
+        - { src: fixes/fix-networking.yml, dest: overcloud-host-configure/pre.d/15-fix-networking.yml }
+        - { src: tools/configure-vxlan.yml, dest: overcloud-host-configure/pre.d/20-configure-vxlan.yml }
+        - { src: tools/growroot.yml, dest: seed-host-configure/pre.d/5-growroot.yml }
+        - { src: fixes/fix-networking.yml, dest: seed-host-configure/pre.d/15-fix-networking.yml }
+        - { src: tools/configure-vxlan.yml, dest: seed-host-configure/pre.d/20-configure-vxlan.yml }
+        - { src: tools/growroot.yml, dest: infra-vm-host-configure/pre.d/5-growroot.yml }
+        - { src: fixes/fix-networking.yml, dest: infra-vm-host-configure/pre.d/15-fix-networking.yml }
+        - { src: tools/configure-vxlan.yml, dest: infra-vm-host-configure/pre.d/20-configure-vxlan.yml }
```

Comment:
As discussed in person, this will break functionality for previous releases. It might be best to include some sort of check to identify the release first, then include the prefix if required